### ISSUE TITLE
[FEATURE]: Automatic change the speed when all the employees have been assigned a job

### ISSUE THREAD

**Issue by ctszkeung** [COMP3122-2324-Project/Simpsyber#43]
Just like in the video, when all the employees have been assigned a job, the player doesn't need to do any operation. So, it will become idel time. If the player forgets to change the speed manually, all the time will be wasted. So change the speed to 2x automatically will facilitae the whole process.

**Comment by alvinlor2001** [comment]
I agree with you completely. It's really frustrating to constantly adjust the speed of the game when there's nothing to do. Alternatively, I think it would be much better if the game slowed back down automatically whenever there was idle staff.
